### DISCUSSION TITLE
Project 64 1.6 won't run on my system, please help..^^

### DISCUSSION THREAD

**Discussion by DonLennox25** [project64/project64#2108] [Support]
Hey there. 
I played a lot of sm64 speedruns on my 1.6 version of pj64 and it worked fine for me all the time. Now after i upgarded my cpu (from ryzen 5 3600x to Ryzen 7 5800x) it won't launch. It tries to launch but crashes immediatly , it wont even show the opening window. I tried reinstall, reboot pc, chekced more compatiblities and tried to open it with sandboxie (it says project64.exe Crashed!).
May someone had the same issue and could help me out ? I just want to grind my speedruns again ^^ ..

**Comment by DerekTurtleRoe** [comment]
@DonLennox25 For speedruns I would recommend a newer version of Project64 since the older versions are very inaccurate and have a lot of issues on modern Windows. We don't provide support for any versions besides the latest stable and the latest development release.
